src/Log/IsoLog.js

```diff
@@ -27,7 +27,7 @@ module.exports = class Log {
 	logs: Array<any>
 	userAgent: string
 
-	constructor() {
+	constructor(options) {
```

Comment:
The constructor is a great place to pass options that are needed at instantiation vs having to call `setOptions` immediately after instantiation.

Having `setOptions` as well is great because it adheres to a good "make sure everything is independently testable" paradigm.

Reply:
Calling `setOptions` is necessary because this library exports `new Log()`: https://github.com/sprucelabsai/sprucebot-log/blob/dev/index.js

The idea being that it can be used immediately without having to instantiate a new logger.

Adding options to the constructor is great for testing though